src/chart/sankey/SankeyView.ts

```diff
@@ -30,6 +30,7 @@ import { RectLike } from 'zrender/src/core/BoundingRect';
 import { setLabelStyle, getLabelStatesModels } from '../../label/labelStyle';
 import { getECData } from '../../util/innerStore';
 import { isString, retrieve3 } from 'zrender/src/core/util';
+import { GraphEdge } from '../../data/Graph';
```

Comment:
It's better if we import the `GraphEdge` type explicitly with the `type` keyword.

Reply:
import with type